src/main/java/com/pikachu/purple/bootstrap/review/dto/request/ReviewUpdateRequest.java

```diff
@@ -0,0 +1,10 @@
+package com.pikachu.purple.bootstrap.review.dto.request;
+
+import jakarta.validation.constraints.Size;
+
+public record ReviewUpdateRequest (
+    Double score,
```

Comment:
p3) PATCH updatr는 이렇게 다 null값 허용해 주어야 합니다!
```suggestion
    @Column(required = false)
    Double score,
```